lib/koala.rb

```diff
@@ -31,24 +34,18 @@ class << self
     # but since the switch to {https://github.com/arsduo/koala/wiki/HTTP-Services Faraday} almost all such goals can be accomplished with middleware.
     attr_accessor :http_service
 
-    def configure
-      yield config
+    def configuration
+      @configuration ||= Configuration.new
```

Comment:
This naming is better, but changing the name could break a whole bunch of apps at once. In order to preserve backwards compatibility, I'm going to keep these but add deprecated accessors for the old names.